array/list-array/script.js

```diff
@@ -34,7 +34,7 @@ function handleSubmit() {
 button.addEventListener("click", function(e) {
   e.preventDefault();
 
-  handleSubmit();
+  handleSubmit
```

Comment:
1. 
```suggestion
  handleSubmit();
```
Ju! De certa forma, essas alterações foram um pouco redundantes. Seguinte, se tu reparar direitinho, adicionasse duas funções ao botão. Você usou o `addEventListener` e também o `onclick`, sendo que, ambos servem para o mesmo propósito: adicionar um comportamento ao elemento e, nesse caso, um comportamento ao ser clicado. Só que é como se você tivesse posto metade do comportamento de um jeito e a outra metade de outro jeito.

Reply:
Oi João, obrigada pela revisão!! :metal:  
Entendi, realmente eu estava chamando duas funções (com o mesmo propósito) de formas diferentes utilizando o `onclick` e `addEventListener`, mas o certo é padronizar essa escrita, até para ficar mais legível e ter um melhor entendimento.